sway/commands/output.c

```diff
@@ -13,6 +13,7 @@ static const struct cmd_handler output_handlers[] = {
 	{ "disable", output_cmd_disable },
 	{ "dpms", output_cmd_dpms },
 	{ "enable", output_cmd_enable },
+	{ "leasing", output_cmd_leasing },
```

Comment:
Obligatory naming bikeshed:

- Should we add "drm" in the name, to make it's clear it's about DRM leasing?
- Should we add an "allow_" prefix, like we have for "allow_tearing"? Might be a bit long and doesn't really improve clarity.

Reply:
I like `allow_drm_leasing`, seems pretty explicit to me